### PR TITLE
cleanup(robot_smach_states)Added docstrings etc. to startup states

### PR DESCRIPTION
**Pull request by jlunenburg**

### CODE REVIEW

File: robot_smach_states/src/robot_smach_states/startup.py [hunk 1]
```diff
@@ -19,9 +19,19 @@ class StartChallengeRobust(smach.StateMachine):
     """Initialize, wait for the door to be opened and drive inside"""
 
     def __init__(self, robot, initial_pose, use_entry_points=False, door=True):
+        """ Initialization method
```
Comment (MatthijsBurgh) [inline robot_smach_states/src/robot_smach_states/startup.py]: I like to start on a new line.

File: robot_smach_states/src/robot_smach_states/startup.py [hunk 2]
```diff
@@ -54,14 +64,16 @@ def __init__(self, robot, initial_pose, use_entry_points=False, door=True):
                                    human_interaction.Say(robot, "Door is open!", block=False),
                                    transitions={"spoken": "INIT_POSE"})
 
-            # Initial pose is set after opening door, otherwise snapmap will fail if door is still closed and initial pose is set,
-            # since it is thinks amigo is standing in front of a wall if door is closed and localization can(/will) be messed up.
+            # Initial pose is set after opening door, otherwise snapmap will fail if door is still closed and initial
+            # pose is set, since it is thinks amigo is standing in front of a wall if door is closed and localization
```
Comment (MatthijsBurgh) [inline robot_smach_states/src/robot_smach_states/startup.py]: amigo....

File: robot_smach_states/src/robot_smach_states/utility.py [hunk 3]
```diff
@@ -10,56 +10,69 @@
 from util.robocup_recorder import start_robocup_recorder
 
 
-# ----------------------------------------------------------------------------------------------------
-
 class Initialize(smach.State):
-    def __init__(self, robot=None):
+    """ Resets the robot (arms, torso, head, etc.) and checks if the tf listener works correctly
```
Comment (MatthijsBurgh) [inline robot_smach_states/src/robot_smach_states/utility.py]: idem, new line.

File: robot_smach_states/src/robot_smach_states/utility.py [hunk 4]
```diff
@@ -149,17 +158,17 @@ class WaitForTrigger(smach.State):
...
         smach.State.__init__(self,
                              outcomes=triggers+['preempted'])
         self.robot = robot
         self.triggers = triggers
+        self.trigger_received = False
 
         # Get the ~private namespace parameters from command line or launch file.
         self.rate = rate
-        topic     = topic
+        topic = topic
```
Comment (MatthijsBurgh) [inline robot_smach_states/src/robot_smach_states/utility.py]: topic = topic ???

File: robot_smach_states/src/robot_smach_states/startup.py (hunk 1 above)
Comment (LoyVanBeek) [reply robot_smach_states/src/robot_smach_states/startup.py]: Let's not clutter and saddle people up with what we like/not like :-)

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings, which says both are fine.

Comment (MatthijsBurgh) [reply robot_smach_states/src/robot_smach_states/startup.py]: It is also inconsistent. Most of the times, the docs start on a new line.

File: robot_smach_states/src/robot_smach_states/utility.py (hunk 3 above)
Comment (jlunenburg) [reply robot_smach_states/src/robot_smach_states/utility.py]: 37d1975b0379d98b5b47905b6ecf0127c75e3c8b

**Comment by jlunenburg** [comment]
@MatthijsBurgh is de moeite niet waard :-P

**Pull request merged**